tools/manual_testing/plot.ipynb

```diff
@@ -0,0 +1,571 @@
+    "    da.coords['yy'] = sc.linspace(dim='yy', unit='km', start=1.1, stop=5.2, num=40)\n",
+    "    da.coords['xx'] = sc.linspace(dim='xx', unit='mm', start=1.1, stop=5.2, num=5)\n",
+    "    da.coords['xx2'] = sc.linspace(dim='xx', unit='m', start=1.1, stop=55.2, num=5)\n",
+    "    da.masks['mask'] = da.coords['xx']['xx',1:] > 5.0 * sc.Unit('mm')\n",
+    "    return da"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "b18a9f17-c8b7-46d5-873d-b2f86f34fefc",
+   "metadata": {},
+   "source": [
+    "# 1d plotting\n",
+    "## General\n",
+    "\n",
+    "Check:\n",
+    "- Axis switching\n",
+    "  - Log buttons remember state\n",
+    "- Slider (before and after switching)\n",
+    "  - Slider range readout\n",
+    "- Profile (before and after switching)\n",
+    "  - Line labels in profile figure\n",
+    "- Keeping lines (check line labels)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "bbe4f783-d24c-4ab4-a31e-f620a7d99894",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "da = make_array()\n",
+    "da['zz',0].plot(projection='1d', vmin=-2*da.unit, vmax=2*da.unit)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e766d02d-aca2-4b5e-9131-c06d83a322a4",
+   "metadata": {},
+   "source": [
+    "Check:\n",
+    "- Similar to above, but with two sliders.\n",
+    "- Profile button should currently be disabled since 2d profile not supported yet."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "d8416058-c17f-4766-ae31-c881802a2fd5",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "da.plot(projection='1d', vmin=-2*da.unit, vmax=2*da.unit)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "36f86c07-cdd8-47ea-a822-58e533ad2d3c",
+   "metadata": {},
+   "source": [
+    "## 1d plot of non-counts\n",
+    "\n",
+    "Check that normalization not messed up, plot should show a constant line at $1~\\text{K}$:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "e8fcdf65-e7f8-47a6-86f4-c1a55973909f",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import scipp as sc\n",
+    "da = sc.DataArray(data=sc.ones(dims=['x'], shape=[100], unit='K'),\n",
+    "                  coords={'x':sc.geomspace(dim='x', start=1, stop=1e6, num=100)})\n",
+    "da.plot()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e316ab2c-9ca3-4631-8aac-d758fca2974d",
+   "metadata": {},
+   "source": [
+    "## 1d plot with multiple lines\n",
+    "\n",
+    "Check:\n",
+    "- Lines can be kept and labels are correct\n",
+    "- Profile works\n",
+    "  - Clicking keeps both lines in profile, with correct labels"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "9fc251de-8813-4642-836c-a3a76fb4e2d7",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "da = make_array()\n",
+    "sc.plot({'a':da['zz',0], 'b':da['zz',1]}, projection='1d')"
+   ]
```

Comment:
Mask shown in wrong place?
<img width="676" alt="image" src="https://user-images.githubusercontent.com/12912489/125070975-f8573080-e0b8-11eb-8cd7-8a450372216d.png">

Reply:
Fixes in latest commit.